org.ioquake3.ioquake3.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: org.ioquake3.ioquake3
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
```suggestion
runtime-version: '23.08'
```

Reply:
Yep, I will do it, but I need to test it first. :-)